installing/install_config/enabling-cgroup-v1.adoc

```diff
@@ -0,0 +1,40 @@
+:_content-type: ASSEMBLY
+:context: nodes-cluster-cgroups-1
+[id="enabling-cgroup-v1"]
+= Enabling Linux control group version 1 (cgroup v1)
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+
+ifndef::openshift-origin[]
+As of {product-title} 4.14, {product-title} uses link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control group version 2] (cgroup v2) in your cluster. If you are using cgroup v1 on {product-title} 4.13 or earlier, migrating to {product-title} 4.14 will not automatically update your cgroup configuration to version 2. A fresh installation of {product-title} 4.14 will use cgroup v2 by default. However, you can enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v1.html[Linux control group version 1] (cgroup v1) upon installation. Enabling cgroup v1 in {product-title} disables all cgroup v2 controllers and hierarchies in your cluster. 
```

Comment:
It looks like this first paragraph in the `ifndef` is the same as the first para in the `ifdef` down on line 19. So you could probably just do the conditionals for the 2nd paragraph only (which does differ)?

Reply:
done